docs/architecture/cx-3-2/edc/policy.definitions.md

```diff
@@ -0,0 +1,272 @@
+# 0. Introduction
```

Comment:
nit: start numbering at 1?

Reply:
The numbering is going to be in flux so let's handle this at the end when the document is complete